styles/styles.css

```diff
@@ -578,3 +578,7 @@ li.dev-name::marker {
     top: 10%;
   }
 }
+
+.line {
```

Comment:
I noticed that the image line is now centered, but the player's tag still shifts to the left/right when there are many cards in the player's deck. Consider wrapping the player's tag in a container and applying a centering method (e.g., flexbox or text alignment) to keep it consistently positioned.